pkg/cluster/api/pdapi.go

```diff
@@ -345,13 +346,23 @@ func (pc *PDClient) EvictStoreLeader(host string, retryOpt *utils.RetryOption) e
 		break
 	}
 
-	if latestStore == nil || latestStore.Status.LeaderCount == 0 {
+	if latestStore == nil {
+		return nil
+	}
+
+	// XXX: the status address in store will be something like 0.0.0.0:20180
+	statusAddress := strings.Split(latestStore.Store.Address, ":")
+	statusAddress[1] = strings.Split(latestStore.Store.StatusAddress, ":")[1]
```

Comment:
Why not use `StatusAddress` directly?

Reply:
Because StatusAddress is something like 0.0.0.0:20180, whose ip address is not correct